Grafik.Logic/JSONHelper.cs

```diff
@@ -34,6 +34,19 @@ public static void ListAllUsers()
                 Console.WriteLine($"{employee.Email} {employee.Name.First} {employee.Name.Last} {employee.Gender}");
             }
         }
-        public static bool CheckIfUserExistsInDatabase(string email)=> _employees.Any(e => e.Email == email);
+        public static bool CheckIfUserExistsInDatabase(string email) => _employees.Any(e => e.Email == email);
+
+        public static Employee SearchForEmployeeByPhoneNumber(string phoneNumber)
+        {
+            
+            try
```

Comment:
w jakim celu jest tutaj ten blok try?

Reply:
First w LINQ wyrzuca error, a nie null, jeżeli nic nie znajdzie, więc muszę to wychwycić

Reply:
w takim przypadku można użyć FirstOrDefault albo można się zastanowić jeżeli oczekujemy że nr telefonu nie będzie się powtarzał a z racji tego że zwracasz Employee to lepiej użyć SingleOrDefault https://docs.microsoft.com/pl-pl/dotnet/api/system.linq.enumerable.singleordefault?view=net-5.0

Reply:
Changed